### DISCUSSION TITLE
RAM size

### DISCUSSION THREAD

**Discussion by gedass2000** [grodansparadis/vscp#259] [General]
How much memory need for VSCP daemon +additional software ? (eq for Raspberry ?)
256 MB ? 512 MB ? 1GB ?
I added one DDR3L chip 256MB to STM32MP153 board.Maybe need to add more ?

![ddr3l](https://user-images.githubusercontent.com/38450851/103747121-d3e7a200-500a-11eb-9f81-5098196a3408.JPG)

**Comment by grodansparadis** [comment]
Strange I commented on this but my comment did not appear to reach here.

256 is working, 512 better. 1G leaves some headroom for node-red and extras as well.  OpenSSL is the most hungry part. 

Can you tell us more about your project. Exciting!

**Reply by grodansparadis** [reply]
Should run on 512M (and even 256M). Need appropriate start up params. though

node-red-pi --max-old-space-size=256

"--max-old-space-size" is recommended to be set to half of available memory.

**Comment by grodansparadis** [comment]
btw built for Italtronic case?

**Comment by grodansparadis** [comment]
I have been looking at Italtronic cases for my upcoming stuff. The drawback is the price.  But the size is good. No experience with Phoenix cases. Any link?

Aluminium cases is really nice. Professional feeling indeed. We used to do them on CC Systems to.  The price is not so good... ;-)  Do you use that for your other CAN module?

**Reply by grodansparadis** [reply]
Sometimes when it comes to DIN enclosed devices it tends top be a lot of boxes. Outer and inner. The inner boxed have no other use then estetic.  Some sort of DIN mounted backplane would be a smarter solution for a modular system. Lower cost to. Or even a plexy glass bottom whih could be mounted to the DIN rail where pcb cards with cutouts where slided into and locked to.
